docs/src/main/tut/repo.md

```diff
@@ -0,0 +1,266 @@
+  case Right(r) => println(r.result)
+}
+```
+
+The `result` on the right is the created [Release][release-scala].
+
+See [the API doc](https://developer.github.com/v3/repos/releases/#create-a-release) for full
+reference.
+
+## Status
+
+### Create a status
+
+You can create a status using `createStatus`, it takes as arguments:
+
+- the repository coordinates (`owner` and `name` of the repository)
+- the `SHA` of the commit for which we want to create a status
+- the state of the status we want to create (can be pending, success, failure or error)
+- other optional parameters: target url, description and context
+
+To create a status:
+
+```tut:silent
+val createStatus =
+  Github(accessToken).repos.createStatus("47deg", "github4s", "aaaaaa", "pending")
+
+createStatus.exec[cats.Id, HttpResponse[String]]() match {
+  case Left(e) => println("Something went wrong: s{e.getMessage}")
+  case Right(r) => println(r.result)
+}
+```
+
+The `result` on the right is the created [Status][status-scala].
+
+See [the API doc](https://developer.github.com/v3/repos/statuses/#create-a-status) for full
+reference.
+
+### List status for a specific Ref
+
+
+You can also list status through `listStatus`, it take as arguments:
+
+- the repository coordinates (`owner` and `name` of the repository)
+- a git ref (a `SHA`, a branch `name` or a tag `name`)
+
+To list the statuses for a specific ref:
+
+```tut:silent
+val listStatus =
+  Github(accessToken).repos.listStatus("47deg", "github4s", "heads/master")
+
+listStatus.exec[cats.Id, HttpResponse[String]]() match {
+  case Left(e) => println("Something went wrong: s{e.getMessage}")
+  case Right(r) => println(r.result)
+}
+```
+
+The `result` on the right is the corresponding [List[Status]][status-scala].
+
+See [the API doc](https://developer.github.com/v3/repos/statuses/#list-statuses-for-a-specific-ref)
+for full reference.
+
+### Get the combined status for a specific Ref
+
+Lastly, you can also get the combined status thanks to `getCombinedStatus`, it takes the same
+arguments as the operation listing statuses:
+
+```tut:silent
+val combinedStatus =
+  Github(accessToken).repos.getCombinedStatus("47deg", "github4s", "heads/master")
+
+combinedStatus.exec[cats.Id, HttpResponse[String]]() match {
+  case Left(e) => println("Something went wrong: s{e.getMessage}")
+  case Right(r) => println(r.result)
+}
+```
+
+The `result` on the right is a [CombinedStatus][status-scala].
+
+Note that the state of the combined status is the product of a heuristic detailed in
+[the API documentation](https://developer.github.com/v3/repos/statuses/#get-the-combined-status-for-a-specific-ref).
+
+As you can see, a few features of the repository endpoint are missing.
+
+As a result, if you'd like to see a feature supported, feel free to create an issue and/or a pull request!
+
+[repository-scala]: https://github.com/47deg/github4s/blob/master/github4s/`SHA`red/src/main/scala/github4s/free/domain/Repository.scala
```

Comment:
the following links are missing / reference nothing:
- status-scala -> repository-scala
- commit-scala -> repository-scala
- user-scala
- release-scala -> repository-scala
- content-scala -> repository-scala